pragma-sdk/pragma_sdk/common/types/currency.py

```diff
@@ -17,8 +17,8 @@ def __init__(
         currency_id: str,
         decimals: Decimals,
         is_abstract_currency: bool,
-        starknet_address: Optional[Address] = None,
-        ethereum_address: Optional[Address] = None,
+        starknet_address: Optional[int | str] = None,
+        ethereum_address: Optional[int | str] = None,
```

Comment:
just change the `Address` type definition then

Reply:
It's fine to have only int, but we can construct it from str or int